src/emulator/gxm/src/gxm.cpp

```diff
@@ -452,39 +455,39 @@ static bool operator<(const emu::SceGxmBlendInfo &a, const emu::SceGxmBlendInfo
     return memcmp(&a, &b, sizeof(a)) < 0;
 }
 
-std::string get_fragment_glsl(SceGxmShaderPatcher &shader_patcher, const SceGxmProgram &fragment_program, const char *base_path) {
+std::string get_fragment_glsl(SceGxmShaderPatcher &shader_patcher, const SceGxmProgram &fragment_program, std::string game_title, const char *shaders_path) {
     const Sha256Hash hash_bytes = sha256(&fragment_program, fragment_program.size);
     const GLSLCache::const_iterator cached = shader_patcher.fragment_glsl_cache.find(hash_bytes);
     if (cached != shader_patcher.fragment_glsl_cache.end()) {
         return cached->second;
     }
 
     const std::array<char, 65> hash_text = hex(hash_bytes);
-    std::string source = load_shader(hash_text.data(), "frag", base_path);
+    std::string source = load_shader(hash_text.data(), "frag", shaders_path);
     if (source.empty()) {
         LOG_ERROR("Missing fragment shader {}", hash_text.data());
         source = generate_fragment_glsl(fragment_program);
-        dump_missing_shader(hash_text.data(), "frag", fragment_program, source.c_str());
+        dump_missing_shader(hash_text.data(), "frag", fragment_program, source.c_str(), shaders_path, game_title);
     }
 
     shader_patcher.fragment_glsl_cache.emplace(hash_bytes, source);
 
     return source;
 }
 
-std::string get_vertex_glsl(SceGxmShaderPatcher &shader_patcher, const SceGxmProgram &vertex_program, const char *base_path) {
+std::string get_vertex_glsl(SceGxmShaderPatcher &shader_patcher, const SceGxmProgram &vertex_program, std::string game_title, const char *shaders_path) {
     const Sha256Hash hash_bytes = sha256(&vertex_program, vertex_program.size);
     const GLSLCache::const_iterator cached = shader_patcher.vertex_glsl_cache.find(hash_bytes);
     if (cached != shader_patcher.vertex_glsl_cache.end()) {
         return cached->second;
     }
 
     const std::array<char, 65> hash_text = hex(hash_bytes);
-    std::string source = load_shader(hash_text.data(), "vert", base_path);
+    std::string source = load_shader(hash_text.data(), "vert", shaders_path);
```

Comment:
I don't think this will find our bundled shaders, because they're in `<base_path>/shaders`.